packages/api/src/controllers/stream.ts

```diff
@@ -447,9 +450,6 @@ export async function getRecordingFields(
   const isUnused = !session.lastSeen && session.createdAt < readyThreshold;
 
   const recordingStatus = isReady ? "ready" : isUnused ? "none" : "waiting";
-  if (!isReady && !forceUrl) {
```

Comment:
This is for the API to return a recordingURL even if it might not have been complete yet. I think this was more useful on the recording v1 to be honest, where recordings were playable even while the stream was live and we only had an artificial wait on the API side to make sure the stream was finished.

I think it would be a noop in this flow here, since the `getRecordingUrls` would return `undefined` anyway and we wouldn't be able to force URLs. Does that seem correct? If so, then it should be completely fine removing this.